libcperciva/events/events_network.c

```diff
@@ -25,7 +25,7 @@
  * the value INT_MAX + 1 (in case every possible file descriptor is in use
```

Comment:
Note to self: there's a missing word in this comment; I'm fixing that in the `libcperciva` repository.